src/tests/compare/test_event_collector.py

```diff
@@ -5,29 +5,31 @@
 from gobupload.compare.event_collector import EventCollector
 
 mock_writer = MagicMock(spec=ContentsWriter)
```

Comment:
maybe rename to contents_writer to make it inline with production code.

Reply:
Done